CRBM-Viz/angular.json

```diff
@@ -28,6 +28,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
I would like to stick to using angualr material only, without pulling in other styles? Would that work?